api/models/app.go

```diff
@@ -61,7 +65,7 @@ func (a1 *App) Equals(a2 *App) bool {
 	// the RHS of && won't eval if eq==false so config checking is lazy
 
 	eq := true
-	eq = eq && a1.Name == a2.Name
+	eq = eq && a1.ID == a2.ID
```

Comment:
name should still be equivalent, too. we're using this code to check if a user actually changed any fields so name is relevant in this context

Reply:
fixed